docker-compose.yml

```diff
@@ -4,7 +4,7 @@ version: '3'
 # Only one of these services should be run at a time.
 services:
   gitlab-ce:
-    image: gitlab/gitlab-ce
+    image: gitlab/gitlab-ce:nightly
```

Comment:
Okay, this doesn't work because the `/version` API doesn't return `-ee` and the `isRunningInEE` checks will fail.

```json
{"version":"14.9.0-pre","revision":"76141b84bb7"}
```